ext/scheduler/airflow2/resources/base_dag.py

```diff
@@ -99,8 +99,51 @@
 )
 {{- end }}
 
+JOB_DIR = "/data"
+IMAGE_PULL_POLICY="IfNotPresent"
+INIT_CONTAINER_IMAGE="optimus:latest" # inject from optimus config ?
```

Comment:
Init container image version should be based on the release version, such that it is compatible with the server deployed.